tomcat/src/main/java/com/linecorp/armeria/server/tomcat/TomcatService.java

```diff
@@ -254,125 +237,21 @@ static String toString(org.apache.catalina.Server server) {
         return buf.toString();
     }
 
-    private final Function<String, Connector> connectorFactory;
-    private final Consumer<Connector> postStopTask;
-    private final ServerListener configurator;
-
-    @Nullable
-    private org.apache.catalina.Server server;
-    @Nullable
-    private Server armeriaServer;
-    @Nullable
-    private String hostname;
-    @Nullable
-    private Connector connector;
-    @Nullable
-    private String engineName;
-    private boolean started;
-
-    private TomcatService(@Nullable String hostname, Function<String, Connector> connectorFactory) {
-        this(hostname, connectorFactory, unused -> { /* unused */ });
```

Comment:
Could you add a package-local default constructor, so that the constructor will not be shown in our Javadoc?